### ISSUE TITLE
I'm an idiot and forgot my password. Probably 3-4 years ago.

### ISSUE THREAD

**Issue by gcbirzan** [pypa/packaging-problems#122]
tl;dr: I'd like to recover the `gcbirzan` account on pypi, if someone isn't squatting on it.

My 'nickname' is generally quite unique, my family name is not incredibly common, nor are my two first names, and it's something I've been using for a while. 

However, I cannot, for the life of me, log in to pypi. I don't remember registering, so someone else might've. I'm not sure what the policy would be for that, but, I really doubt in the tech world (or anywhere else, to be honest) you'd find another gcbirzan. I'm willing to provide ID proof (privately, of course).

Bottom line. There's a gcbirzan account on pypi that I cannot access. If nothing else, I'd like to know if it was made by me, I guess by the email. But, ideally, I'd like to 'recover' that account. 

P.S. Blame @jonparrott for this ticket. :(

**Comment by ewdurbin** [comment]
@gcbirzan email ernest at python dot org and we'll try to get you sorted.
